index.html

```diff
@@ -6,6 +6,22 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <link rel="icon" href="/favicon.ico" />
     <title>Cuttle</title>
+
+    
+    <meta property="og:url" content="https://www.cuttle.cards/" />
```

Comment:
We need some sort of env for the site url instead of hardcoding it.

Reply:
@seriouslysean I created #462 to track this

Reply:
@seriouslysean I created #462 to track this